src/game.cpp

```diff
@@ -5250,22 +5250,54 @@ void game::control_vehicle()
             veh->start_engines( true );
         }
     } else {
-        const cata::optional<tripoint> examp_ = choose_adjacent( _( "Control vehicle where?" ) );
-        if( !examp_ ) {
-            return;
+        std::vector<optional_vpart_position> vehicles;
+        tripoint sole_position;
+        for( const tripoint elem : m.points_in_radius( g->u.pos(), 1 ) ) {
+            if( const optional_vpart_position vp = m.veh_at( elem ) ) {
+                const cata::optional<vpart_reference> vpr = vp.value().part_with_feature( "CONTROLS", true );
+                if( vpr ) {
+                    vehicles.push_back( vp );
+                    sole_position = elem;
+                }
+            }
         }
-        const optional_vpart_position vp = m.veh_at( *examp_ );
-        if( !vp ) {
-            add_msg( _( "No vehicle there." ) );
+        if( vehicles.size() < 1 ) {
+            add_msg( _( "No vehicle controls in reach." ) );
```

Comment:
This implies there are vehicle controls *somewhere* just not in reach - is that intentional? (I suck at reading c++, pls ignore if that's not the case) Does `No vehicle controls found` make more sense?